### ISSUE TITLE
Can't use more than one filter with new RestAPI

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-client#16]
```
# linstor sp l -n pve1 -s data
root@linstor2-controller-0:/# linstor sp l -n pve1 -s data
ERROR:
    The given node name 'pve1storage_pools=data' is invalid.

# linstor --curl sp l -n pve1 -s data
curl -X GET http://localhost:3370/v1/view/storage-pools?nodes=pve1storage_pools=data
```

Looks like `&`  is skipped

**Comment by rp-** [comment]
what version do you use? this was fixed with python-linstor 0.9.5:
https://github.com/LINBIT/linstor-api-py/commit/13d5bbd10d3bbac4fe5c934aa5e9ab7361c9cd2b

**Comment by kvaps** [comment]
Agree latest version is working fine
```
linstor 0.9.6; GIT-hash: c7c973dc39c93e8d45d2b7e3a0907b1d20fb3dfe
```
